### PR TITLE
billing devtool and docs

### PR DESCRIPTION
**Pull request by pjain1**
Fixes https://github.com/rilldata/rill-private-issues/issues/714

This does not include instructions on how to use utility cmd that @AdityaHegde created to manipulate end dates in billing issues metadata to simulate various scenarios. May be we can move the cmds into the project and add instructions in readme or point to that repo but that would be private repo.

### CODE REVIEW

File: cli/cmd/devtool/data/cloud-deps.docker-compose.yml [hunk 1]
```diff
@@ -50,3 +50,8 @@ services:
       - "55679:55679" # zpages extension
     depends_on:
       - zipkin
+  stripe-cli:
+    image: stripe/stripe-cli:latest
+    command: listen --forward-to http://host.docker.internal:8080/payment/webhook
+    environment:
+      - STRIPE_API_KEY=
```
Comment (pjain1) [inline cli/cmd/devtool/data/cloud-deps.docker-compose.yml:57]: need to discuss this

File: cli/cmd/devtool/data/template.env
Comment (pjain1) [inline cli/cmd/devtool/data/template.env:1]: This file seems unused, not sure where it is used.

File: cli/cmd/billing/subscription/subscription.go [hunk 2]
```diff
@@ -13,7 +13,6 @@ func SubscriptionCmd(ch *cmdutil.Helper) *cobra.Command {
 	}
 
 	subsCmd.PersistentFlags().StringVar(&ch.Org, "org", ch.Org, "Organization Name")
-	subsCmd.AddCommand(CreateCmd(ch))
```
Comment (pjain1) [inline cli/cmd/billing/subscription/subscription.go:16]: Edit cmd does the same thing

File: cli/cmd/devtool/start.go [hunk 3]
```diff
@@ -710,6 +717,39 @@ func (s local) awaitUI(ctx context.Context) error {
...
+func prepareStripeConfig() error {
+	templateFile := "cli/cmd/devtool/data/stripe-config.template"
+	outputDir := "dev-cloud-state"
+	outputFile := filepath.Join(outputDir, "stripe-config.toml")
+
+	apiKey := lookupDotenv("RILL_DEVTOOL_STRIPE_CLI_API_KEY")
+
+	if apiKey == "" {
+		logWarn.Printf("No Stripe API key found in .env, Stripe webhook events will not be processed\n")
+	}
```
Comment (begelundmuller) [inline cli/cmd/devtool/start.go:729]: Should there be a `return nil` here?

**Review by begelundmuller — APPROVED** [review]

File: cli/cmd/devtool/start.go (hunk 3 above)
Comment (pjain1) [reply cli/cmd/devtool/start.go:729]: What do you think, just didn't wanted to block devtool in case of missing api key

Comment (pjain1) [reply cli/cmd/devtool/start.go:729]: oh misread the comment, wanted to create the file anyways so that docker compose volume thing does not fail

Comment (begelundmuller) [reply cli/cmd/devtool/start.go:729]: Got it :)

**Pull request merged**